src/wrapper.test.js

```diff
@@ -319,39 +269,6 @@ describe('Distro initialization', () => {
     });
   });
 
-  describe('On Amazon EKS', () => {
```

Comment:
don't we need this test?

Reply:
No, we removed the collector. The test did not go red because the test harness was broken until now.